doc/api/cli.md

```diff
@@ -2252,6 +2257,39 @@ This option may be specified multiple times to include multiple glob patterns.
 If both `--test-coverage-exclude` and `--test-coverage-include` are provided,
 files must meet **both** criteria to be included in the coverage report.
 
+### `--test-coverage-branches=threshold`
```

Comment:
These flags appear to be in the wrong sorting order relative to `--test-coverage-exclude` and `--test-coverage-include`.